src/pages/Pools/index.tsx

```diff
@@ -206,15 +206,15 @@ export default function Pools() {
 
   const { loading: loadingUserLpPositions, data: userLpPairs } = useLPPairs(account || undefined)
 
-  const handleCurrencySelect = useCallback(token => {
-    setFilterToken(token as Token)
+  const handleCurrencySelect = useCallback((token: any) => {
```

Comment:
This `any` should be `Token`. Right?

Reply:
Yes it should be a Token but where ever it was used its not Token thats why I am casting it. But its a bad practice.
Will do another PR to fix this. I have some other cases also to fix.